content/package.json

```diff
@@ -15,14 +15,19 @@
         "eslint:fix": "eslint . --fix",
         "lint": "npm-run-all --parallel eslint stylelint",
         "stylelint": "stylelint '**/*.css' '**/*.less'",
-        "stylelint:fix": "stylelint '**/*.css' '**/*.less' --fix"
+        "stylelint:fix": "stylelint '**/*.css' '**/*.less' --fix",
+        "build": "clientlib --verbose"
     },
     "devDependencies": {
+        "aem-clientlib-generator": "^1.7.2",
         "aemfed": "^0.1.1",
         "eslint": "^4.19.1",
         "npm-run-all": "^4.1.5",
         "stylelint": "^9.10.1",
         "stylelint-config-recommended": "^2.1.0",
         "stylelint-selector-bem-pattern": "^2.0.0"
+    },
+    "dependencies": {
+        "@adobe/adobe-client-data-layer": "latest"
```

Comment:
To avoid confusion I would be explicitly define the real version (e.g. 1.0.0), so that we always know what exactly we include in the clientlib.